WeakAuras/GenericTrigger.lua

```diff
@@ -4049,7 +4049,7 @@ function GenericTrigger.GetAdditionalProperties(data, triggernum)
       elseif type(v.enable) == "boolean" then
         enable = v.enable
       end
-      if (enable and v.store and v.name and v.display) then
+      if (enable and v.store and v.name and v.display and v.conditionType ~= "bool") then
```

Comment:
the "type" is actually the "display-gui widget" type, whereas the "conditionType" is actually closer to what's stored in the state, so using the conditionType is better.